.github/workflows/githib-actions.yml

```diff
@@ -0,0 +1,44 @@
+name: Java CI with Gradle
+
+on: [push, pull_request]
+
+jobs:
+    job:
+        name: "${{ matrix.os_label }}"
+        runs-on: ${{ matrix.os }}
+        strategy:
+            fail-fast: false
+            matrix:
+                name: [ ubuntu, macos ]
+                include:
+                    - name: ubuntu
+                      os: ubuntu-latest
+                      os_label: Ubuntu
+
+                    - name: macos
+                      os: macos-latest
+                      os_label: MacOS
+
+        steps:
+            -   uses: actions/checkout@v2
+            -   name: Set up JDK 11
+                uses: actions/setup-java@v2
+                with:
+                    java-version: '11'
+                    distribution: 'adopt'
+                    cache: gradle
+
+            -   name: Grant execute permission for gradlew
+                run: chmod +x gradlew
```

Comment:
Why do you grant permissions here? Why not to commit gradlew with necessary permissions in the repository?

Reply:
Fixed in #5 PR